main/inc/lib/AnnouncementManager.php

```diff
@@ -1384,7 +1384,9 @@ public static function edit_announcement_attachment_file($id_attach, $file, $fil
             } else {
                 $new_file_name = uniqid('');
                 $new_path = $updir . '/' . $new_file_name;
-                @move_uploaded_file($file['tmp_name'], $new_path);
+		
+		// This file is copy here but its cleaned in api_mail_html in api.lib.php    
+                copy($file['tmp_name'], $new_path);
```

Comment:
Minor comment: the change does not *only* contain a comment or style change, so don't put "Minor" in the commit message ;-)